.github/workflows/test.yml

```diff
@@ -32,3 +32,30 @@ jobs:
           go get -v -t -d ./...
       - name: Test
         run: go test ./...
+
+  android-client:
+    name: Android Client Test
+    runs-on: ubuntu-latest
+    services:
+      threads:
+        image: textile/go-threads
+        ports:
+        - 6006:6006
```

Comment:
this is going to cause CI issues when there are breaking API/pb changes in a PR (so this release image will be out of sync)

Reply:
not following what you're saying here.

Reply:
ah it's a docker image?

Reply:
why don't we just spin up the go-threads daemon from the current git revision that the CI is operating on?

Reply:
could do, i was doing that in js-threads-client for a bit it just felt hacky

Reply:
Doesn't sound like a hack to me, sounds like the right way to do how. It's the only way you can guarantee that the server and client are running the same versions.

Reply:
or, I should say "compatible" versions.

Reply:
yeah, i was wondering if, since the dockerfile is here in the branch maybe i could do the same i'm doing from the local definition... but not sure there is a way for that

Reply:
this https://github.community/t5/GitHub-Actions/Docker-action-in-same-repository-with-Dockerfile-in-root/td-p/38757 didn't seem to do the trick for me